Cargo.toml

```diff
@@ -63,12 +63,23 @@ gethostname = "0.5"
 hex-literal = "0.4"
 
 [features]
-default = ["signature-pgp"]
+default = [
+    "signature-pgp",
+    "gzip-compression",
+    "zstd-compression",
+    "xz-compression",
+    "bzip2-compression"
```

Comment:
To be perfectly honest it might be reasonable to leave bzip2 out by default. It's relatively uncommon and isn't that great at compressing things in most circumstances compared to modern alternatives.